perfkitbenchmarker/alicloud/ali_disk.py

```diff
@@ -131,4 +134,27 @@ def Detach(self):
 
   def GetDevicePath(self):
     """Returns the path to the device inside the VM."""
-    return '/dev/xvd%s' % self.device_letter
+    if FLAGS.ali_io_optimized == 'none':
```

Comment:
If you change https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/alicloud/ali_virtual_machine.py#L43 from 'none' to be None then you can write the code list this:
if FLAGS.ali_io_optimized is not None:]
...

or if you dont list the negative switch the then and else causes for
if FLAGS.ali_io_optimized:
  return '/dev/vd%s' % self.device_letter
else:
  return '/dev/xvd%s' % self.device_letter